package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "recharts",
-  "version": "2.10.3",
+  "version": "2.10.4",
```

Comment:
Please don't bump the version unless an official recharts release, we'll handle this internally

Reply:
Of course, I'll revert this.